### ISSUE TITLE
Project maintainer should also be a user

### ISSUE THREAD

**Issue by JustinLove** [JustinLove/autoscaler#56]
I've stopped running the project that I built autoscaler for.  I'm out of touch with real world issues, don't have a realistic test environment, and generally don't have a big incentive to keep up with things.  Having a more active maintainer would be good.

**Comment by devill** [comment]
Is this still the case? My project heavily relies on this gem, so I'm willing to take on the responsibility of managing pull requests and releases.

**Comment by JustinLove** [comment]
Yes, it's still the case - Things are pretty quiet at the moment but I'm relying on tests and sometimes the sample app to validate changes, and I'm not in a position to experience any routine issues.

**Comment by devill** [comment]
In the meantime I actually stopped using the gem. :(
